e2e/basic/docker-compose.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 services:
   envoy:
-    image: envoyproxy/envoy:v1.31-latest
```

Comment:
This needs to be `v1.32+` for the updated wasmtime, which means istio `v1.24+` (`sched_yield` missing in v1.31)